jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/WatchServiceTest.java

```diff
@@ -94,7 +94,7 @@ void setUp(@TempDir Path tempDir) throws IOException {
         .project(JavaProject.builder()
           .baseDirectory(tempDir.toFile())
           .build())
-        .outputDirectory(Files.createDirectory(tempDir.resolve("target")).toFile().getAbsolutePath())
+        .outputDirectory(Files.createDirectory(tempDir.resolve("target")).toFile().getName())
```

Comment:
Looks good.
However, for clarity, we might want to create the target directory in a variable outside the builder and then just reference it here.

```java
Files.createDirectory(tempDir.resolve("target"));
// suggested changes
```
```suggestion
        .outputDirectory("target")
```